tests/reporter/kubernetes.go

```diff
@@ -973,7 +973,7 @@ func (r *KubernetesReporter) dumpK8sEntityToFile(virtCli kubecli.KubevirtClient,
 
 	response, err := virtCli.RestClient().Get().RequestURI(requestURI).Do(context.Background()).Raw()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to dump entity named [%s]: %v\n", entityName, err)
+		fmt.Fprintf(os.Stderr, "dump %s: %v\n", entityName, err)
```

Comment:
everything else looks good to me but I feel like this new text doesn't convey that it's a failure

Reply:
Well, my point is that in most of the cases (except of the very limited sriov lanes) this is *not* a failure. As it stands, it keeps distracting me when I search for `fail` in order to understand why a test lane failed, so I try to make it stand out less. Still, it is legible that something is amiss:
```
dump sriovoperatorconfigs: the server could not find the requested resource
```
Do you have a better text that does not use `fail`?

Ideally, each lane (or set of tests) would declare which CRDs they need to dump. Alas, we do not use pytest where doing this kind of things is trivial.

Reply:
seeing what the text would be does make it clear it's a failure, thanks :-)<